xcp_d/workflow/anatomical.py

```diff
@@ -269,1218 +297,1136 @@ def init_anatomical_wf(
+                "template_to_t1w_xform",
+                "lh_inflated_surf",
+                "rh_inflated_surf",
+                "lh_midthickness_surf",
+                "rh_midthickness_surf",
+                "lh_pial_surf",
+                "rh_pial_surf",
+                "lh_smoothwm_surf",
+                "rh_smoothwm_surf",
+            ],
+        ),
+        name="inputnode",
+    )
 
-    mnitemplate = get_template(template="MNI152NLin6Asym", resolution=2, desc=None, suffix="T1w")
+    outputnode = pe.Node(
+        niu.IdentityInterface(
+            fields=[
+                "lh_inflated_surf",
+                "rh_inflated_surf",
+                "lh_midthickness_surf",
+                "rh_midthickness_surf",
+                "lh_pial_surf",
+                "rh_pial_surf",
+                "lh_smoothwm_surf",
+                "rh_smoothwm_surf",
+            ],
+        ),
+        name="outputnode",
+    )
 
-    if input_type in ("dcan", "hcp"):
-        # TODO: Replace with layout.get call(s). No reason to search through a list of strings.
-        all_files = list(layout.get_files())
-        L_inflated_surf = fnmatch.filter(
-            all_files, "*sub-*" + subject_id + "*hemi-L_inflated.surf.gii"
-        )[0]
-        R_inflated_surf = fnmatch.filter(
-            all_files, "*sub-*" + subject_id + "*hemi-R_inflated.surf.gii"
-        )[0]
-        L_midthick_surf = fnmatch.filter(
-            all_files, "*sub-*" + subject_id + "*hemi-L_midthickness.surf.gii"
-        )[0]
-        R_midthick_surf = fnmatch.filter(
-            all_files, "*sub-*" + subject_id + "*hemi-R_midthickness.surf.gii"
-        )[0]
-        L_pial_surf = fnmatch.filter(all_files, "*sub-*" + subject_id + "*hemi-L_pial.surf.gii")[0]
-        R_pial_surf = fnmatch.filter(all_files, "*sub-*" + subject_id + "*hemi-R_pial.surf.gii")[0]
-        L_wm_surf = fnmatch.filter(all_files, "*sub-*" + subject_id + "*hemi-L_smoothwm.surf.gii")[
-            0
-        ]
-        R_wm_surf = fnmatch.filter(all_files, "*sub-*" + subject_id + "*hemi-R_smoothwm.surf.gii")[
-            0
-        ]
+    if not warp_to_standard:
+        merge_files_to_list = pe.Node(
+            niu.Merge(8),
+            name="merge_files_to_list",
+        )
 
-        # All of the converted dcan and hcp files should have a session entity/folder
-        ses_id = _getsesid(R_wm_surf)
-        anatdir = os.path.join(output_dir, "xcp_d", f"sub-{subject_id}", f"ses-{ses_id}", "anat")
-        os.makedirs(anatdir, exist_ok=True)
-
-        surf = [
-            L_inflated_surf,
-            R_inflated_surf,
-            L_midthick_surf,
-            R_midthick_surf,
-            L_pial_surf,
-            R_pial_surf,
-            L_wm_surf,
-            R_wm_surf,
-        ]
+        # fmt:off
+        workflow.connect([
+            (inputnode, merge_files_to_list, [
+                ("lh_inflated_surf", "in1"),
+                ("rh_inflated_surf", "in2"),

```

Comment:
@madisoth AFAICT, the inflated surfaces aren't warped to standard space, and aren't used for anything except naming HCP-style output files generated later in the workflow. Is there a reason these files are included?

Reply:
Correct, they're only used for naming the HCP-style outputs

Originally we did warp them to standard space but it often fail due to the surfaces extending outside the FOV of the warp.

Reply:
Ah, I see. In that case, we don't really need them at all, since we know how the HCP-style inflated files should be named based on other surfaces. I will remove the regular inflated files entirely.

Reply:
Well, we do need to write out the inflated files to the derivatives folder _if_ they're already in standard space and are (presumably) HCP-style, so I will limit them instead of removing them.